packages/composite-checkout/src/components/checkout.js

```diff
@@ -67,7 +69,7 @@ export default function Checkout( {
 	return (
 		<Container className={ joinClasses( [ className, 'checkout' ] ) }>
 			<MainContent className={ joinClasses( [ className, 'checkout__content' ] ) }>
-				{ OrderSummary && <OrderSummaryStep OrderSummary={ OrderSummary } /> }
+				<OrderSummaryStep OrderSummary={ OrderSummary ? OrderSummary : CheckoutOrderSummary } />
```

Comment:
Nice! This can be shortened a bit to `<OrderSummaryStep OrderSummary={ OrderSummary || CheckoutOrderSummary } />`